tests/ROCrate/ISAProfile/Person.Tests.fs

```diff
@@ -14,7 +14,7 @@ let mandatory_properties = Person(
 let all_properties = Person(
```

Comment:
How are these tests still passing when you deleted all the types tested? maybe not seeing the forest for the trees here